### ISSUE TITLE
deployment crashes when includes multibyte in paths?

### ISSUE THREAD

**Issue by sawanoboly** [netlify/cli#30]
**- Do you want to request a *feature* or report a *bug*?**

probably bug.

**- What is the current behavior?**

`netlify deploy` fails to deployment.

```
/usr/bin/netlify deploy --site-id xxxxx --path yyyyyyyyy --access-token xxxxxxxxxxxxxxxxx --draft
Deploying folder: yyyyyyyyyyyyy
[                                        ] UploadingPotentially unhandled rejection [2] Authentication failed (WARNING: non-Error used)
Potentially unhandled rejection [4] Authentication failed (WARNING: non-Error used)
Potentially unhandled rejection [6] Authentication failed (WARNING: non-Error used)
Potentially unhandled rejection [8] Authentication failed (WARNING: non-Error used)
Potentially unhandled rejection [10] Authentication failed (WARNING: non-Error used)
Potentially unhandled rejection [12] Authentication failed (WARNING: non-Error used)
[==                                      ] UploadingPotentially unhandled rejection [14] Authentication failed (WARNING: non-Error used)
Potentially unhandled rejection [16] Authentication failed (WARNING: non-Error used)
Potentially unhandled rejection [18] Authentication failed (WARNING: non-Error used)
Potentially unhandled rejection [20] Authentication failed (WARNING: non-Error used)
/ ProcessingPotentially unhandled rejection [22] Authentication failed (WARNING: non-Error used)
Potentially unhandled rejection [24] Authentication failed (WARNING: non-Error used)
Potentially unhandled rejection [26] Authentication failed (WARNING: non-Error used)
Potentially unhandled rejection [28] Authentication failed (WARNING: non-Error used)
Potentially unhandled rejection [30] Authentication failed (WARNING: non-Error used)
Potentially unhandled rejection [32] Authentication failed (WARNING: non-Error used)
Potentially unhandled rejection [34] Authentication failed (WARNING: non-Error used)
Potentially unhandled rejection [36] Authentication failed (WARNING: non-Error used)
Potentially unhandled rejection [38] Authentication failed (WARNING: non-Error used)
| ProcessingPotentially unhandled rejection [40] Authentication failed (WARNING: non-Error used)
Potentially unhandled rejection [42] Authentication failed (WARNING: non-Error used)
Potentially unhandled rejection [44] Authentication failed (WARNING: non-Error used)
Potentially unhandled rejection [46] Authentication failed (WARNING: non-Error used)
Potentially unhandled rejection [48] Authentication failed (WARNING: non-Error used)
\ ProcessingPotentially unhandled rejection [50] Authentication failed (WARNING: non-Error used)
- ProcessingPotentially unhandled rejection [52] Authentication failed (WARNING: non-Error used)
Authentication failed
Error during deploy:  Authentication failed
```

**- If the current behavior is a bug, please provide the steps to reproduce.**

1. create directory which has index.html and there are subdirectories named with multibyte language. each subdirectories has index.html and some static files.
2. run `netlify deploy`

**- What is the expected behavior?**

deployment without errors until complete.

**- Please mention your node.js, and operating system version.**

```
node -v
v4.3.2

netlify --version
1.2.2

cat /etc/os-release 
NAME="Alpine Linux"
ID=alpine
VERSION_ID=3.3.3
PRETTY_NAME="Alpine Linux v3.3"
```

**Comment by mabdullah353** [comment]
```
$ netlify deploy dist -e development -t $ACCESS_TOKEN_NETLIFY
Deploying folder: ./build
[1000D[K[                                        ] UploadingPotentially unhandled rejection [2] Authentication failed (WARNING: non-Error used)
Potentially unhandled rejection [4] Authentication failed (WARNING: non-Error used)
Potentially unhandled rejection [6] Authentication failed (WARNING: non-Error used)
Potentially unhandled rejection [8] Authentication failed (WARNING: non-Error used)
Potentially unhandled rejection [10] Authentication failed (WARNING: non-Error used)
Potentially unhandled rejection [12] Authentication failed (WARNING: non-Error used)
Potentially unhandled rejection [14] Authentication failed (WARNING: non-Error used)
```
Getting the following error any solution to it ?

**Comment by sawanoboly** [comment]
@mabdullah353 FYI.

There is another implementation by Go language. It might work.
https://github.com/netlify/netlifyctl

**Comment by bcomnes** [comment]
We are currently assessing how to update the node cli.  The go cli is in better shape at the moment.

**Comment by bcomnes** [comment]
This may be addressed by the upcoming 2.0.0.  Leaving open so we can verify.

**Comment by bcomnes** [comment]
I believe this has been addressed now. We can reopen if someone finds a new reproducible case.
